<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE log SYSTEM "logger.dtd">
<log>
<record>
  <date>2016-02-12T13:54:22</date>
  <millis>1455281662305</millis>
  <sequence>4352</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>INFO</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>30</thread>
  <message>Cache has been refresh for view: listNgi</message>
</record>
<record>
  <date>2016-02-12T13:54:22</date>
  <millis>1455281662399</millis>
  <sequence>4382</sequence>
  <logger>org.glassfish.grizzly.http.server.NetworkListener</logger>
  <level>INFO</level>
  <class>org.glassfish.grizzly.http.server.NetworkListener</class>
  <method>start</method>
  <thread>16</thread>
  <message>Started listener bound to [0.0.0.0:8080]</message>
</record>
<record>
  <date>2016-02-12T13:54:22</date>
  <millis>1455281662406</millis>
  <sequence>4383</sequence>
  <logger>org.glassfish.grizzly.http.server.HttpServer</logger>
  <level>INFO</level>
  <class>org.glassfish.grizzly.http.server.HttpServer</class>
  <method>start</method>
  <thread>16</thread>
  <message>[HttpServer] Started.</message>
</record>
<record>
  <date>2016-02-12T13:54:23</date>
  <millis>1455281663132</millis>
  <sequence>4507</sequence>
  <logger>class fr.in2p3.lavoisier.engine.view.ViewAbstract</logger>
  <level>WARNING</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewAbstract</class>
  <method>createChain</method>
  <thread>29</thread>
  <message>Entering fallback because of error [Exception raised for view 'customFactorsRaw' [java.io.IOException: 404 page not found
]]</message>
</record>
<record>
  <date>2016-02-12T13:54:23</date>
  <millis>1455281663134</millis>
  <sequence>4508</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>WARNING</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>29</thread>
  <message>Failed to refresh cache for view: customFactorsRaw</message>
  <exception>
    <message>fr.in2p3.lavoisier.interfaces.error.InitializationException: Exception raised for view 'customFactorsRaw' [java.io.IOException: 404 page not found
]</message>
    <frame>
      <class>fr.in2p3.lavoisier.chaining.AdaptorConfigurator</class>
      <method>getConfiguredAdaptor</method>
      <line>42</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.chaining.ChainImpl</class>
      <method>&lt;init&gt;</method>
      <line>21</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.engine.ChainOfAdaptors</class>
      <method>&lt;init&gt;</method>
      <line>46</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.engine.view.ViewAbstract</class>
      <method>createChain</method>
      <line>117</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
      <method>refresh</method>
      <line>149</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.engine.task.CacheRefreshTask</class>
      <method>run</method>
      <line>22</line>
    </frame>
    <frame>
      <class>java.util.concurrent.ThreadPoolExecutor</class>
      <method>runWorker</method>
      <line>1145</line>
    </frame>
    <frame>
      <class>java.util.concurrent.ThreadPoolExecutor$Worker</class>
      <method>run</method>
      <line>615</line>
    </frame>
    <frame>
      <class>java.lang.Thread</class>
      <method>run</method>
      <line>745</line>
    </frame>
  </exception>
</record>
<record>
  <date>2016-02-12T13:54:23</date>
  <millis>1455281663178</millis>
  <sequence>4541</sequence>
  <logger>org.glassfish.grizzly.http.server.NetworkListener</logger>
  <level>INFO</level>
  <class>org.glassfish.grizzly.http.server.NetworkListener</class>
  <method>start</method>
  <thread>16</thread>
  <message>Started listener bound to [0.0.0.0:443]</message>
</record>
<record>
  <date>2016-02-12T13:54:23</date>
  <millis>1455281663179</millis>
  <sequence>4542</sequence>
  <logger>org.glassfish.grizzly.http.server.HttpServer</logger>
  <level>INFO</level>
  <class>org.glassfish.grizzly.http.server.HttpServer</class>
  <method>start</method>
  <thread>16</thread>
  <message>[HttpServer-1] Started.</message>
</record>
<record>
  <date>2016-02-12T13:54:23</date>
  <millis>1455281663261</millis>
  <sequence>4545</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>INFO</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>31</thread>
  <message>Cache has been refresh for view: goc.ServiceFlavours</message>
</record>
<record>
  <date>2016-02-12T13:54:23</date>
  <millis>1455281663361</millis>
  <sequence>4692</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>INFO</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>31</thread>
  <message>Cache has been refresh for view: listSite</message>
</record>
<record>
  <date>2016-02-12T13:54:23</date>
  <millis>1455281663534</millis>
  <sequence>4810</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>INFO</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>32</thread>
  <message>Cache has been refresh for view: ngiContactsList</message>
</record>
<record>
  <date>2016-02-12T13:54:23</date>
  <millis>1455281663702</millis>
  <sequence>5000</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>WARNING</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>32</thread>
  <message>Failed to refresh cache for view: customFactors</message>
  <exception>
    <message>fr.in2p3.lavoisier.interfaces.error.AdaptorException: Exception raised for view 'customFactorsRaw' [Adaptor raised exception -- java.io.FileNotFoundException: /tmp/argo/customFactorsRaw/current]</message>
    <frame>
      <class>fr.in2p3.lavoisier.chaining.link.connector.SAXConnectorLink</class>
      <method>generate</method>
      <line>26</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.chaining.Chain</class>
      <method>generate</method>
      <line>120</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.chaining.Chain</class>
      <method>writeToContentHandler</method>
      <line>103</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
      <method>writeToXMLEventHandler</method>
      <line>117</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.xml.XmlView</class>
      <method>write</method>
      <line>54</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.connector.XMLConnector</class>
      <method>writeToContentHandler</method>
      <line>34</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.chaining.link.connector.SAXConnectorLink</class>
      <method>generate</method>
      <line>24</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.chaining.Chain</class>
      <method>generate</method>
      <line>120</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.chaining.Chain</class>
      <method>writeToCache</method>
      <line>109</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
      <method>refresh</method>
      <line>151</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.engine.task.CacheRefreshTask</class>
      <method>run</method>
      <line>22</line>
    </frame>
    <frame>
      <class>java.util.concurrent.ThreadPoolExecutor</class>
      <method>runWorker</method>
      <line>1145</line>
    </frame>
    <frame>
      <class>java.util.concurrent.ThreadPoolExecutor$Worker</class>
      <method>run</method>
      <line>615</line>
    </frame>
    <frame>
      <class>java.lang.Thread</class>
      <method>run</method>
      <line>745</line>
    </frame>
  </exception>
</record>
<record>
  <date>2016-02-12T13:54:23</date>
  <millis>1455281663857</millis>
  <sequence>5196</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>INFO</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>31</thread>
  <message>Cache has been refresh for view: navbar</message>
</record>
<record>
  <date>2016-02-12T13:54:24</date>
  <millis>1455281664118</millis>
  <sequence>6462</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>INFO</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>32</thread>
  <message>Cache has been refresh for view: aggregation_profiles</message>
</record>
<record>
  <date>2016-02-12T13:54:24</date>
  <millis>1455281664123</millis>
  <sequence>6464</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>INFO</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>30</thread>
  <message>Cache has been refresh for view: goc.siteList</message>
</record>
<record>
  <date>2016-02-12T13:54:24</date>
  <millis>1455281664191</millis>
  <sequence>6626</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>INFO</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>30</thread>
  <message>Cache has been refresh for view: default</message>
</record>
<record>
  <date>2016-02-12T13:54:24</date>
  <millis>1455281664293</millis>
  <sequence>6662</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>INFO</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>31</thread>
  <message>Cache has been refresh for view: reports</message>
</record>
<record>
  <date>2016-02-12T13:54:24</date>
  <millis>1455281664322</millis>
  <sequence>6691</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>INFO</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>31</thread>
  <message>Cache has been refresh for view: categories</message>
</record>
<record>
  <date>2016-02-12T13:54:24</date>
  <millis>1455281664390</millis>
  <sequence>6797</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>INFO</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>31</thread>
  <message>Cache has been refresh for view: _default_</message>
</record>
<record>
  <date>2016-02-12T13:54:24</date>
  <millis>1455281664841</millis>
  <sequence>6800</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>INFO</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>32</thread>
  <message>Cache has been refresh for view: comparison_cern_view_cloud</message>
</record>
<record>
  <date>2016-02-12T13:54:25</date>
  <millis>1455281665271</millis>
  <sequence>6801</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>INFO</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>30</thread>
  <message>Cache has been refresh for view: goc.HostList</message>
</record>
<record>
  <date>2016-02-12T13:54:25</date>
  <millis>1455281665963</millis>
  <sequence>6803</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>INFO</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>29</thread>
  <message>Cache has been refresh for view: comparison_cern_view</message>
</record>
<record>
  <date>2016-02-12T13:54:26</date>
  <millis>1455281666329</millis>
  <sequence>6875</sequence>
  <logger>class fr.in2p3.lavoisier.engine.view.ViewAbstract</logger>
  <level>WARNING</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewAbstract</class>
  <method>createChain</method>
  <thread>29</thread>
  <message>Entering fallback because of error [Exception raised for view 'comparison_main' [java.io.IOException: 404 page not found
]]</message>
</record>
<record>
  <date>2016-02-12T13:54:26</date>
  <millis>1455281666329</millis>
  <sequence>6874</sequence>
  <logger>class fr.in2p3.lavoisier.engine.view.ViewAbstract</logger>
  <level>WARNING</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewAbstract</class>
  <method>createChain</method>
  <thread>31</thread>
  <message>Entering fallback because of error [Exception raised for view 'comparison_main_cloud' [java.io.IOException: 404 page not found
]]</message>
</record>
<record>
  <date>2016-02-12T13:54:26</date>
  <millis>1455281666331</millis>
  <sequence>6876</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>WARNING</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>29</thread>
  <message>Failed to refresh cache for view: comparison_main</message>
  <exception>
    <message>fr.in2p3.lavoisier.interfaces.error.InitializationException: Exception raised for view 'comparison_main' [java.io.IOException: 404 page not found
]</message>
    <frame>
      <class>fr.in2p3.lavoisier.chaining.AdaptorConfigurator</class>
      <method>getConfiguredAdaptor</method>
      <line>42</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.chaining.ChainImpl</class>
      <method>&lt;init&gt;</method>
      <line>21</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.engine.ChainOfAdaptors</class>
      <method>&lt;init&gt;</method>
      <line>46</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.engine.view.ViewAbstract</class>
      <method>createChain</method>
      <line>117</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
      <method>refresh</method>
      <line>149</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.engine.task.CacheRefreshTask</class>
      <method>run</method>
      <line>22</line>
    </frame>
    <frame>
      <class>java.util.concurrent.ThreadPoolExecutor</class>
      <method>runWorker</method>
      <line>1145</line>
    </frame>
    <frame>
      <class>java.util.concurrent.ThreadPoolExecutor$Worker</class>
      <method>run</method>
      <line>615</line>
    </frame>
    <frame>
      <class>java.lang.Thread</class>
      <method>run</method>
      <line>745</line>
    </frame>
  </exception>
</record>
<record>
  <date>2016-02-12T13:54:26</date>
  <millis>1455281666334</millis>
  <sequence>6877</sequence>
  <logger>fr.in2p3.lavoisier.engine.view.ViewCached</logger>
  <level>WARNING</level>
  <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
  <method>refresh</method>
  <thread>31</thread>
  <message>Failed to refresh cache for view: comparison_main_cloud</message>
  <exception>
    <message>fr.in2p3.lavoisier.interfaces.error.InitializationException: Exception raised for view 'comparison_main_cloud' [java.io.IOException: 404 page not found
]</message>
    <frame>
      <class>fr.in2p3.lavoisier.chaining.AdaptorConfigurator</class>
      <method>getConfiguredAdaptor</method>
      <line>42</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.chaining.ChainImpl</class>
      <method>&lt;init&gt;</method>
      <line>21</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.engine.ChainOfAdaptors</class>
      <method>&lt;init&gt;</method>
      <line>46</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.engine.view.ViewAbstract</class>
      <method>createChain</method>
      <line>117</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.engine.view.ViewCached</class>
      <method>refresh</method>
      <line>149</line>
    </frame>
    <frame>
      <class>fr.in2p3.lavoisier.engine.task.CacheRefreshTask</class>
      <method>run</method>
      <line>22</line>
    </frame>
    <frame>
      <class>java.util.concurrent.ThreadPoolExecutor</class>
      <method>runWorker</method>
      <line>1145</line>
    </frame>
    <frame>
      <class>java.util.concurrent.ThreadPoolExecutor$Worker</class>
      <method>run</method>
      <line>615</line>
    </frame>
    <frame>
      <class>java.lang.Thread</class>
      <method>run</method>
      <line>745</line>
    </frame>
  </exception>
</record>
<record>
  <date>2016-02-12T13:55:19</date>
  <millis>1455281719846</millis>
  <sequence>23960</sequence>
  <logger>fr.in2p3.lavoisier.connector.RotatingCacheConnector</logger>
  <level>WARNING</level>
  <class>fr.in2p3.lavoisier.connector.RotatingCacheConnector</class>
  <method>writeToContentHandler</method>
  <thread>39</thread>
  <message>RotatingCacheConnector has deleted unreferenced file: /tmp/argo/api.status.rotating.caches/1455281539488</message>
</record>
